colly.go

```diff
@@ -462,6 +464,12 @@ func CheckHead() CollectorOption {
 	}
 }
 
+func FixCharset(f bool) CollectorOption {
+	return func(c *Collector) {
+		c.fixCharset = f
+	}
+}
+
```

Comment:
How is this different from `DetectCharset` option?

Reply:
if fixCharset  set false,  user can fix charset onResponse